packages/sandpack-core/src/resolver/fixture/node_modules/its-fine/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "its-fine",
+  "version": "1.0.8",
+  "description": "A collection of escape hatches for React.",
+  "keywords": [
+    "react",
+    "fiber",
+    "internal",
+    "reconciler",
+    "hooks"
+  ],
+  "author": "Cody Bennett (https://github.com/codyjasonbennett)",
+  "maintainers": [
+    "Paul Henschel (https://github.com/drcmda)"
+  ],
+  "homepage": "https://github.com/pmndrs/its-fine",
+  "repository": "https://github.com/pmndrs/its-fine",
+  "license": "MIT",
+  "files": [
+    "out/*",
+    "src/*"
+  ],
+  "type": "module",
+  "types": "./out/index.d.ts",
+  "main": "./out/index.cjs",
+  "module": "./out/index.js",
+  "react-native": "./out/index.js",
+  "exports": {
+    "types": "./out/index.d.ts",
+    "require": "./out/index.cjs",
+    "import": "./out/index.js"
+  },
```

Comment:
1. what is requested for this package?
2. what does it resolve to right now?
3. what does it resolve to with this patch?

Reply:
Currently it resolves to nothing, gonna debug it now

Reply:
Seems like the resolver did not handle the exports correctly if it's a root exports, omitting the `.` was not possible.

Thanks for pointing to the webpack docs btw, really nice reference for this

Reply:
Oh, it makes sense that it didn't work for `exports: Record<Condition, FilePath>`. Nice catch! I assume that a regular shorthand, like this `exports: './file.js'`, works OK, right?

Reply:
Yes the regular shorthand worked fine and was also tested, but didn't know the root `Record<Condition, FilePath>` was a thing, so didn't implement it